src/Schema/AbstractSchemaManager.php

```diff
@@ -677,6 +677,33 @@ public function dropAndCreateView(View $view)
         $this->createView($view);
     }
 
+    /* alterSchema() Methods */
```

Comment:
This is redundant. We should get rid of the remaining annotations like this later.